agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -1411,6 +1411,29 @@ bool slave_thread::handle_cmdu_backhaul_manager_message(
 
         break;
     }
+    case beerocks_message::ACTION_BACKHAUL_HOSTAP_STATS_MEASUREMENT_REQUEST: {
+        LOG(DEBUG) << "ACTION_BACKHAUL_HOSTAP_STATS_MEASUREMENT_REQUEST";
+        if (monitor_socket) {
```

Comment:
According to our coding standards:
```suggestion
        if (!monitor_socket) {
            break;
        }
        ...        
```